src/components/tag-skill/tag-skill.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import iconTick from '../../svgs/icon-tick.svg';
+import Icon from '../icon/icon.jsx';
+import Tag from '../tag/tag.jsx';
+import styles from './tag-skill.css';
+
+export default function TagSkill(props) {
+  return (
+    <Tag readOnly>
```

Comment:
I believe we now need `a11yKey`

Reply:
This should be captured in _some_ failing test because we have a prop-type violation. Does a test currently fail? Otherwise, I think it would be beneficial to look into getting it to fail -- I thought I did set it up.